tests/resources/hooks/oldhook.js

```diff
@@ -1,5 +1,7 @@
+'use strict';
+
 exports.cliVersion = '1.0';
 
 exports.init = function (logger, config, cli, appc) {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/resources/h...
  
  
  DangerID: danger-id-default;
  File: tests/resources/hooks/oldhook.js;
  Line: 5;
-->

- :warning: tests/resources/hooks/oldhook.js line 5 – 'appc' is defined but never used. (no-unused-vars)